src/util/sss_ini.c

```diff
@@ -588,6 +588,39 @@ int sss_confdb_create_ldif(TALLOC_CTX *mem_ctx,
 }
 
 #ifdef HAVE_LIBINI_CONFIG_V1_3
+static errno_t check_domain_inherit_from(char *cfg_section,
+                                         struct ini_cfgobj *config_obj,
+                                         struct ini_errobj *errobj)
+{
+    struct value_obj *vo = NULL;
+    int ret;
+
+    ret = ini_get_config_valueobj(cfg_section,
+                                  "inherit_from",
+                                  config_obj,
+                                  INI_GET_NEXT_VALUE,
+                                  &vo);
+    if (ret != EOK) {
+        goto done;
+    }
+
+    if (vo != NULL) {
+        ret = ini_errobj_add_msg(errobj,
+                                 "Attribute 'inherit_from' is not "
+                                 "allowed in section '%s'. Check for "
```

Comment:
I agree with you, but I just relocated this piece of code and this message existed beforehand. I didn't want to change it in case some test or user script is grepping for it.
Do you think it is safe to change it?

Reply:
I checked our test suite and I do not see any test affected by such change. So I think we should change it.

Thanks
Tom

Reply:
Done